ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_manipulation.py

```diff
@@ -1015,3 +1015,51 @@ def test_expand(
         x=x[0],
         shape=shape,
     )
+
+
+@st.composite
+def _concat_from_sequence_helper(draw):
+    shape = draw(st.shared(helpers.get_shape(min_num_dims=1), key="values_shape"))
+    num_arrays = draw(
+        st.shared(helpers.ints(min_value=1, max_value=3), key="num_arrays")
+    )
+    dtype = draw(st.sampled_from(draw(helpers.get_dtypes("valid"))))
+    arrays = []
+    dtypes = [dtype for _ in range(num_arrays)]
+
+    for _ in range(num_arrays):
+        array = draw(helpers.array_values(dtype=dtype, shape=shape))
+        arrays.append(np.asarray(array, dtype=dtype))
+    return dtypes, arrays
+
+
+# concat_from_sequence
+@handle_test(
+    fn_tree="functional.ivy.experimental.concat_from_sequence",
+    dtypes_arrays=_concat_from_sequence_helper(),
+    new_axis=st.integers(min_value=0, max_value=1),
+    axis=st.integers(min_value=0, max_value=1),
```

Comment:
`axis` is not necessarily 0 or 1. Its value depends on the shape of the input arrays. You should generate this in the composite strategy above. You can use the `get_axis` helper.